July-Morning/assignment1/Assignment_1_Q2.cpp

```diff
@@ -0,0 +1,104 @@
+#include <iostream>
+
+typedef struct node

```

Comment:
In C++ you can write structs using the same style as class, no need for old C style with typedef:

```C++
struct node {
    int data;
    node* next;
};
```